src/NSubstitute/Core/Extensions.cs

```diff
@@ -25,6 +25,11 @@ public static string Join(this IEnumerable<string> strings, string separator)
             return string.Join(separator, strings);
         }
 
+        public static bool IsDelegate(this Type type)
+        {
+            return type.GetTypeInfo().IsSubclassOf(typeof(MulticastDelegate));
```

Comment:
Original was just `typeof(Delegate)`.  Any reason for change to `MulticastDelegate`? (don't think it makes a difference in practice, just curious.)

Reply:
I left comment in the code to describe reasoning about that. Good catch - it's a good practice to leave comment when you do something non-obvious 🙏